server/settings.py

```diff
@@ -9,3 +9,5 @@
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 SECRET_KEY = os.environ.get('SECRET_KEY')
 OMDB_API_KEY = os.environ.get('OMDB_API_KEY')
+JWT_ACCESS_LIFESPAN = {'days': 1}
```

Comment:
the default lifespan was 15 minutes, so I made the default lifespan 24 hours so access tokens last longer before needing to be refrehsed